cmake/Conan.cmake

```diff
@@ -4,10 +4,10 @@
 
 if (NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
   message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
-  file(DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/v0.16.1/conan.cmake"
-       "${CMAKE_BINARY_DIR}/conan.cmake"
-       EXPECTED_HASH SHA256=396e16d0f5eabdc6a14afddbcfff62a54a7ee75c6da23f32f7a31bc85db23484
-       TLS_VERIFY ON)
+  file(DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/0.18.1/conan.cmake"
```

Comment:
Did you clear your cmake folders and conan folder inside your system to make sure this version works.

Conan folder: C:\Users\<user>\.conan\data

Reply:
I only deleted the cmake-folders initialy.
Retried with both \data and cmake folders deleted.
All test passed (debug & release)
Formatting still working in logging sandbox.